Base/usr/share/man/man2/getresuid.md

```diff
@@ -0,0 +1,34 @@
+## Name
+
+getresuid, getresgid - get real, effective, and saved user / group id
+
+## Synopsis
+
+```**c++
+#include <unistd.h>
+
+int getresuid(uid_t *, uid_t *, uid_t *);
+int getresgid(gid_t *, gid_t *, gid_t *);
```

Comment:
The Serenity style is to put the pointer star (and reference ampersand) next to the basic type, and man pages should follow the same style.

Reply:
Done. It was wrong i the code too, fixed there as well.